cromwell_tools/cromwell_auth.py

```diff
@@ -59,19 +60,19 @@ def _validate_auth(header, auth):
 
     @staticmethod
     def _validate_url(url):
-        """Validate the input Cromwell url and harmonize it.
+        """Validate the input Cromwell url and harmonize it. Python 2/3 compatible.
```

Comment:
I don't think it's necessary to say this is `Python 2/3 compatible` since the package is supposed to be compatible with both.

Reply:
Also since "harmonize" could mean many things, how about changing this to "Validate the input Cromwell url and remove the trailing slash"?

Reply:
Good points!